patches/minecraft/net/minecraft/client/renderer/EntityRenderer.java.patch

```diff
@@ -8,6 +8,15 @@
          }
      }
  
+@@ -290,7 +291,7 @@
+             this.field_78531_r.func_175607_a(this.field_78531_r.field_71439_g);
+         }
+ 
+-        float f3 = this.field_78531_r.field_71441_e.func_175724_o(new BlockPos(this.field_78531_r.func_175606_aa()));
++        float f3 = this.field_78531_r.field_71441_e.func_175724_o(getBlockPosAtEyeHeight());
```

Comment:
You could use `getPositionEyes` like this:
```java
float f3 = this.mc.world.getLightBrightness(new BlockPos(this.mc.getRenderViewEntity().getPositionEyes(1f)));
```
which would save having to patch in an extra function.